test/spec_helper.rb

```diff
@@ -52,15 +52,6 @@ def require(path)
 
   # track all ruby files under src
   SimpleCov.track_files("#{SRC_PATH}/lib/**/*.rb")
-
```

Comment:
Are we losing coveralls? Or is it still used?

Reply:
The coveralls needs to be moved to a different place when running parallel tests. The old approach sent the coverage at the end when collecting the coverage statistics. But that does not work properly when running the tests in parallel, each parallel process would submit a separate partial coverage only for the executed tests.

We need to send the coverage just once, after *all* parallel threads (processes) are finished.

The coverage is sent directly from `rake test:unit` after all parallel tests are finished. See
https://github.com/yast/yast-rake/blob/master/lib/tasks/test_unit.rake#L29-L35

For the backward compatibility when using the standard sequential RSpec the coverage is sent in the old way (from `spec/test_helper.rb`). It would be better to unify this, but then we would need to touch ~100 packages...